File: ueueo-framework/ueueo-uow/src/main/java/com/ueueo/uow/NullUnitOfWorkEventPublisher.cs
using System.Collections.Generic;
using System.Threading.Tasks;
using Volo.Abp.DependencyInjection;

namespace Volo.Abp.Uow;

public class NullUnitOfWorkEventPublisher : IUnitOfWorkEventPublisher, ISingletonDependency
{
    public void PublishLocalEventsAsync(IEnumerable<UnitOfWorkEventRecord> localEvents)
    {
        return Task.CompletedTask;
    }

    public void PublishDistributedEventsAsync(IEnumerable<UnitOfWorkEventRecord> distributedEvents)
    {
        return Task.CompletedTask;
    }
}

File: ueueo-framework/ueueo-uow/src/main/java/com/ueueo/uow/NullUnitOfWorkEventPublisher.java
package com.ueueo.uow;

import java.util.Collection;

public class NullUnitOfWorkEventPublisher implements IUnitOfWorkEventPublisher {
    @Override
    public void publishLocalEvents(Collection<UnitOfWorkEventRecord> localEvents) {

    }

    @Override
    public void publishDistributedEvents(Collection<UnitOfWorkEventRecord> distributedEvents) {

    }
}
